utils/db_utils.py

```diff
@@ -94,7 +114,26 @@ def query_confirmed_trips(start_date: str, end_date: str, tz: str):
         # Add primary modes from the sensed, inferred and ble summaries. Note that we do this
         # **before** filtering the `all_trip_columns` because the
         # *_section_summary columns are not currently valid
-        get_max_mode_from_summary = lambda md: max(md["distance"], key=md["distance"].get) if len(md["distance"]) > 0 else "INVALID"
+        
+        # Check if 'md' is not a dictionary or does not contain the key 'distance'
```

Comment:
@TeachMeTW shouldn't this have already been included in https://github.com/e-mission/op-admin-dashboard/pull/121

Reply:
That's what I thought so too so I'm not sure why it was not transferred. I synced and pulled but still showed this.